src/dune_rules/dune_load.ml

```diff
@@ -155,7 +155,7 @@ module Script = struct
     let* () =
       let* (_ : Memo.Run.t) = Memo.current_run () in
       Memo.of_reproducible_fiber
-        (Process.run Strict ~display:!Clflags.display ~dir:(Path.source dir)
+        (Process.run Strict ~display:Quiet ~dir:(Path.source dir)
```

Comment:
Here is a call to ocamlc for generating a Dune file. This is internal and does not need to be reported to the user.